server/errorHandler.test.ts

```diff
@@ -14,7 +14,7 @@ beforeEach(() => {
   app = appWithAllRoutes({
     services: {
       dataAccess: {} as DataAccess,
-      commonApiRoutes: {} as CommonApiRoutes,
+      commonApiRoutes: commonApiRoutesMock(),
```

Comment:
i had to introduce this mock because there were a few undefined request handlers in the `/api/*/image` routes that didn’t matter before as they didn’t reach express when wrapped with the async middleware. but now they are added to a real router and need to exist.